agent/fishing_agent.py

```
@@ -308,6 +337,65 @@ def _calculate_blue_click_timing(
+                if n >= 40:
+                    released_green = True
+                    break
+                if time.time() - t0 > max_hold:
+                    try:
+                        from PIL import Image
+                        dbg = os.path.join(
+                            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+                            "debug", "fishing_cast_debug.png")
+                        Image.fromarray(img[..., :3][..., ::-1]).save(dbg)
+                        print(f"    🐞 已保存蓄力中调试帧: {dbg}")
+                    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)